src/arraymath.jl

```diff
@@ -163,3 +163,11 @@ end
         return a
     end
 end
+
+Base.min{T1<:Real,T2<:Real}(a::StaticArray{T1}, b::StaticArray{T2}) = _min(promote(a, b)...)
```

Comment:
Hmm... the definition of `min` here is just plain *odd*. I see now that `Base` defines this, but wouldn't `min.(a, b)` work better and semantically make *much* more sense? And codegen for this is already great with *StaticArrays*.

Personally, I see this as one of the autovectorization operations that accidentally has not been pruned from `Base` yet. Can whatever method that calls this be updated to use the dot-call syntax?